egs/aspire/s5/run.sh

```diff
@@ -160,7 +161,7 @@ steps/train_sat.sh  --cmd "$train_cmd" \
 local/build_silprob.sh
 
 # train the neural network model
-local/multi_condition/run_nnet2_ms.sh
+local/multi_condition/run_nnet2_ms.sh --aspire-data $aspire_data
```

Comment:
Could you also please call local/chain/run_tdnn.sh rather than the nnet2 script.